### ISSUE TITLE
Alchemy is not rewarding

### ISSUE THREAD

**Issue by mickvangelderen** [mickvangelderen/graveyard-keeper-issue-tracker#12]
https://steamcommunity.com/app/599140/discussions/4/1734336452565543348/

The downside of failed experimentation is so big that it forces you to look up recipes online. The game itself should make the recipes available to you somehow instead.

**Comment by mickvangelderen** [comment]
https://steamcommunity.com/app/599140/discussions/0/1734336452558333157/

> The Alchemy tier 2 system is hilarious. Anyone done the maths behind a 3 ingredient combinataion of around 10 possible ingredients? To 'brute force' you through this is extensively dull and super stupid.
I would also refer to a wiki for the tier 2 recipes. With a 2-combination system it is manageable but still rather dull. I have done this systematically with a beautiful google spreadsheet and reloading to minimize the goo-production, since you have no clue what to do with the goo in any part of the game and burning it (destroy) is no option.
> The alchemy system is nothing to "understand" and nothing you can "reason" about. No clues, no hints.Plain nothing. Why would "Alcohol + Slowing Solution" make a White Paint? Or "Health Powder + Chaos Solution = Heal Potion"? To me this looks totally random.
This is a rather lame element in the game which eats up the fun.

> TL;DR: google it to keep the fun.
